test/md2test.c

```diff
@@ -10,7 +10,6 @@
 #include <string.h>
 
 #include "../e_os.h"
```

Comment:
should we just add -I.. to all flags and fix e_os.h include's while you're here touching everything?